app/src/main/java/com/teamltt/carcare/fragment/MyObdResponseRecyclerViewAdapter.java

```diff
@@ -53,8 +53,8 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     public void onBindViewHolder(final ViewHolder holder, int position) {
```

Comment:
This is used by HomeActivity, TripsActivity, and ResponseFragment. I believe TripsActivity was added for demo purposes and I'm not sure if it's still accessible. After looking at HomeActivity, line 98, it appears that this Adapter is still being instantiated, however I can't find anywhere where data is being added to it. I believe it's safe to delete it.